app/helpers/admin_helper.rb

```diff
@@ -3,6 +3,8 @@ module AdminHelper
   # side navbar in alphabetical order.
   # If you add an item before "config", please update the insert call in
   # admin_menu_items below.
+
+  # @ridhwana to delete these when we move over to new admin layout
```

Comment:
Putting in these comments looks and feel a bit weird but it will make it much easier to find and delete the necessary code next week when we move over to the new layout.